tests/integration/test_interaction_assessment.py

```diff
@@ -474,13 +474,29 @@ def load_scenario(self, explanation_text: str, scenario_id: int):
         self._add_scenario(scenario_page_id, scenario_page_title, scenario_xml)
         self._page = self.go_to_page(scenario_page_title)
 
+    @staticmethod
+    def _get_explanation_html(explanation: str):
+        """
+        get generated html explanation
```

Comment:
Nit: the sentence in the docstring should start with an uppercase letter and end with a full stop.